SofaKernel/modules/Sofa.Config/CMakeLists.txt

```diff
@@ -203,7 +205,10 @@ if(WIN32)
             # remove lots of warnings (the Wall of "normal" clang seems different of Wall of clang-cl)
             list(APPEND SOFACONFIG_COMPILE_OPTIONS -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-double-promotion -Wno-old-style-cast -Wno-reserved-id-macro -Wno-language-extension-token -Wno-dllexport-explicit-instantiation-decl -Wno-nonportable-system-include-path -Wno-zero-as-null-pointer-constant -Wno-documentation)
             # optimization flags (not sure if necessary..)
-            list(APPEND SOFACONFIG_COMPILE_OPTIONS_RELEASE /fp:fast -march=native)
+            list(APPEND SOFACONFIG_COMPILE_OPTIONS_RELEASE -march=native)
```

Comment:
`march=native` means it is not possible to package SOFA for another processor than the one used to compile it.
I know Clang compilation on Windows is experimental. I just wanted to clarify it.